docs/reference/commandline/help.md

```diff
@@ -0,0 +1,71 @@
+# help
+
+<!---MARKER_GEN_START-->
+Help about the command
+
+### Subcommands
+
+| Name | Description |
+| --- | --- |
+| [`attach`](attach.md) | Attach local standard input, output, and error streams to a running container |
```

Comment:
@thaJeztah Looks like we missed a case for subcommands in https://github.com/docker/cli-docs-tool/pull/34

Reply:
@crazy-max that's weird; all other commands do have the table properly aligned, except for `help`? 🤔

Reply:
Could it be using Cobra's built-in markdown generator? I don't see `help` in the list of things generated;

<details>

```
+ /tmp/docsgen --formats md --source /go/src/github.com/docker/cli/docs/reference/commandline --target /go/src/github.com/docker/cli/docs/reference/commandline
INFO: Generating Markdown for "docker attach"
INFO: Generating Markdown for "docker build"
INFO: Generating Markdown for "docker builder build"
INFO: Generating Markdown for "docker builder prune"
INFO: Generating Markdown for "docker builder"
INFO: Generating Markdown for "docker checkpoint create"
INFO: Generating Markdown for "docker checkpoint ls"
INFO: Generating Markdown for "docker checkpoint rm"
INFO: Generating Markdown for "docker checkpoint"
INFO: Generating Markdown for "docker commit"
INFO: Generating Markdown for "docker config create"
INFO: Generating Markdown for "docker config inspect"
INFO: Generating Markdown for "docker config ls"
INFO: Generating Markdown for "docker config rm"
INFO: Generating Markdown for "docker config"
INFO: Generating Markdown for "docker container attach"
INFO: Generating Markdown for "docker container commit"
INFO: Generating Markdown for "docker container cp"
INFO: Generating Markdown for "docker container create"
INFO: Generating Markdown for "docker container diff"
INFO: Generating Markdown for "docker container exec"
INFO: Generating Markdown for "docker container export"
INFO: Generating Markdown for "docker container inspect"
INFO: Generating Markdown for "docker container kill"
INFO: Generating Markdown for "docker container logs"
INFO: Generating Markdown for "docker container ls"
INFO: Generating Markdown for "docker container pause"
INFO: Generating Markdown for "docker container port"
INFO: Generating Markdown for "docker container prune"
INFO: Generating Markdown for "docker container rename"
INFO: Generating Markdown for "docker container restart"
INFO: Generating Markdown for "docker container rm"
INFO: Generating Markdown for "docker container run"
INFO: Generating Markdown for "docker container start"
INFO: Generating Markdown for "docker container stats"
INFO: Generating Markdown for "docker container stop"
INFO: Generating Markdown for "docker container top"
INFO: Generating Markdown for "docker container unpause"
INFO: Generating Markdown for "docker container update"
INFO: Generating Markdown for "docker container wait"
INFO: Generating Markdown for "docker container"
INFO: Generating Markdown for "docker context create"
INFO: Generating Markdown for "docker context export"
INFO: Generating Markdown for "docker context import"
INFO: Generating Markdown for "docker context inspect"
INFO: Generating Markdown for "docker context ls"
INFO: Generating Markdown for "docker context rm"
INFO: Generating Markdown for "docker context show"
INFO: Generating Markdown for "docker context update"
INFO: Generating Markdown for "docker context use"
INFO: Generating Markdown for "docker context"
INFO: Generating Markdown for "docker cp"
INFO: Generating Markdown for "docker create"
INFO: Generating Markdown for "docker diff"
INFO: Generating Markdown for "docker events"
INFO: Generating Markdown for "docker exec"
INFO: Generating Markdown for "docker export"
INFO: Generating Markdown for "docker history"
INFO: Generating Markdown for "docker image build"
INFO: Generating Markdown for "docker image history"
INFO: Generating Markdown for "docker image import"
INFO: Generating Markdown for "docker image inspect"
INFO: Generating Markdown for "docker image load"
INFO: Generating Markdown for "docker image ls"
INFO: Generating Markdown for "docker image prune"
INFO: Generating Markdown for "docker image pull"
INFO: Generating Markdown for "docker image push"
INFO: Generating Markdown for "docker image rm"
INFO: Generating Markdown for "docker image save"
INFO: Generating Markdown for "docker image tag"
INFO: Generating Markdown for "docker image"
INFO: Generating Markdown for "docker images"
INFO: Generating Markdown for "docker import"
INFO: Generating Markdown for "docker info"
INFO: Generating Markdown for "docker inspect"
INFO: Generating Markdown for "docker kill"
INFO: Generating Markdown for "docker load"
INFO: Generating Markdown for "docker login"
INFO: Generating Markdown for "docker logout"
INFO: Generating Markdown for "docker logs"
INFO: Generating Markdown for "docker manifest annotate"
INFO: Generating Markdown for "docker manifest create"
INFO: Generating Markdown for "docker manifest inspect"
INFO: Generating Markdown for "docker manifest push"
INFO: Generating Markdown for "docker manifest rm"
INFO: Generating Markdown for "docker manifest"
INFO: Generating Markdown for "docker network connect"
INFO: Generating Markdown for "docker network create"
INFO: Generating Markdown for "docker network disconnect"
INFO: Generating Markdown for "docker network inspect"
INFO: Generating Markdown for "docker network ls"
INFO: Generating Markdown for "docker network prune"
INFO: Generating Markdown for "docker network rm"
INFO: Generating Markdown for "docker network"
INFO: Generating Markdown for "docker node demote"
INFO: Generating Markdown for "docker node inspect"
INFO: Generating Markdown for "docker node ls"
INFO: Generating Markdown for "docker node promote"
INFO: Generating Markdown for "docker node ps"
INFO: Generating Markdown for "docker node rm"
INFO: Generating Markdown for "docker node update"
INFO: Generating Markdown for "docker node"
INFO: Generating Markdown for "docker pause"
INFO: Generating Markdown for "docker plugin create"
INFO: Generating Markdown for "docker plugin disable"
INFO: Generating Markdown for "docker plugin enable"
INFO: Generating Markdown for "docker plugin inspect"
INFO: Generating Markdown for "docker plugin install"
INFO: Generating Markdown for "docker plugin ls"
INFO: Generating Markdown for "docker plugin push"
INFO: Generating Markdown for "docker plugin rm"
INFO: Generating Markdown for "docker plugin set"
INFO: Generating Markdown for "docker plugin upgrade"
INFO: Generating Markdown for "docker plugin"
INFO: Generating Markdown for "docker port"
INFO: Generating Markdown for "docker ps"
INFO: Generating Markdown for "docker pull"
INFO: Generating Markdown for "docker push"
INFO: Generating Markdown for "docker rename"
INFO: Generating Markdown for "docker restart"
INFO: Generating Markdown for "docker rm"
INFO: Generating Markdown for "docker rmi"
INFO: Generating Markdown for "docker run"
INFO: Generating Markdown for "docker save"
INFO: Generating Markdown for "docker search"
INFO: Generating Markdown for "docker secret create"
INFO: Generating Markdown for "docker secret inspect"
INFO: Generating Markdown for "docker secret ls"
INFO: Generating Markdown for "docker secret rm"
INFO: Generating Markdown for "docker secret"
INFO: Generating Markdown for "docker service create"
INFO: Generating Markdown for "docker service inspect"
INFO: Generating Markdown for "docker service logs"
INFO: Generating Markdown for "docker service ls"
INFO: Generating Markdown for "docker service ps"
INFO: Generating Markdown for "docker service rm"
INFO: Generating Markdown for "docker service rollback"
INFO: Generating Markdown for "docker service scale"
INFO: Generating Markdown for "docker service update"
INFO: Generating Markdown for "docker service"
INFO: Generating Markdown for "docker stack config"
INFO: Generating Markdown for "docker stack deploy"
INFO: Generating Markdown for "docker stack ls"
INFO: Generating Markdown for "docker stack ps"
INFO: Generating Markdown for "docker stack rm"
INFO: Generating Markdown for "docker stack services"
INFO: Generating Markdown for "docker stack"
INFO: Generating Markdown for "docker start"
INFO: Generating Markdown for "docker stats"
INFO: Generating Markdown for "docker stop"
INFO: Generating Markdown for "docker swarm ca"
INFO: Generating Markdown for "docker swarm init"
INFO: Generating Markdown for "docker swarm join"
INFO: Generating Markdown for "docker swarm join-token"
INFO: Generating Markdown for "docker swarm leave"
INFO: Generating Markdown for "docker swarm unlock"
INFO: Generating Markdown for "docker swarm unlock-key"
INFO: Generating Markdown for "docker swarm update"
INFO: Generating Markdown for "docker swarm"
INFO: Generating Markdown for "docker system df"
INFO: Generating Markdown for "docker system dial-stdio"
INFO: Generating Markdown for "docker system events"
INFO: Generating Markdown for "docker system info"
INFO: Generating Markdown for "docker system prune"
INFO: Generating Markdown for "docker system"
INFO: Generating Markdown for "docker tag"
INFO: Generating Markdown for "docker top"
INFO: Generating Markdown for "docker trust inspect"
INFO: Generating Markdown for "docker trust key generate"
INFO: Generating Markdown for "docker trust key load"
INFO: Generating Markdown for "docker trust key"
INFO: Generating Markdown for "docker trust revoke"
INFO: Generating Markdown for "docker trust sign"
INFO: Generating Markdown for "docker trust signer add"
INFO: Generating Markdown for "docker trust signer remove"
INFO: Generating Markdown for "docker trust signer"
INFO: Generating Markdown for "docker trust"
INFO: Generating Markdown for "docker unpause"
INFO: Generating Markdown for "docker update"
INFO: Generating Markdown for "docker version"
INFO: Generating Markdown for "docker volume create"
INFO: Generating Markdown for "docker volume inspect"
INFO: Generating Markdown for "docker volume ls"
INFO: Generating Markdown for "docker volume prune"
INFO: Generating Markdown for "docker volume rm"
INFO: Generating Markdown for "docker volume update"
INFO: Generating Markdown for "docker volume"
INFO: Generating Markdown for "docker wait"
INFO: Generating Markdown for "docker"
+ clean
+ rm -rf /tmp/docker-cli-docsgen.ZpFcN9cDf8
```

</details>

Reply:
Ah good point yeah, looks handled by Cobra. I guess it's fine then.

Reply:
I'm actually wondering if we should skip generating for the `help` command; we don't print it on the `--help` output (I think), and we never advertised it (I think it's one of those automatic features from Cobra). 🤔